src/Kis/Time.hs

```diff
@@ -11,13 +13,23 @@ import System.Time.Extra
 
 newtype TimeOffset = TimeOffset DiffTime
 
+data ClockConfig = RealTime | VirtualTime Double
```

Comment:
Ah sorry, du hast es ja in der PR-Beschreibung auch geschrieben. Ich bin mir nicht ganz sicher, ob das jetzt schon so wichtig ist.
Ich glaube eher, dass wir Clock erst mal irgendwie parametrisieren sollten, so dass es nicht zwangsläufig in IO laufen muss, sondern einfach in der gleichen Monade wie das Kis läuft.